x-pack/plugins/security_solution/server/lib/detection_engine/signals/bulk_create_threshold_signals.ts

```diff
@@ -146,13 +107,11 @@ const getTransformedHits = (
               field,
               value: bucket.key,
             },
-          ],
-          cardinality: !isEmpty(threshold.cardinality_field)
+          ].filter((term) => term.field != null),
```

Comment:
for follow up: `field` is typed as `string` in the function definition but used as `string | null` - looks like `aggParts` is being typed as `any` so we're losing type checking there